client/app/components/Resizable/index.jsx

```diff
@@ -51,7 +51,7 @@ export default function Resizable({ toggleShortcut, direction, sizeAttribute, ch
 
   const resizeHandle = useMemo(
     () => (
-      // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+      // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
```

Comment:
For what it's worth, I'm pretty sure we add keyboard events at a higher level, so we shouldn't feel bad about this.

Reply:
The linter rules will bug us about it. Shall I plain remove them?

Reply:
I think you can leave it for now, LGTM.